frontend/README.md

```
@@ -2,16 +2,24 @@ This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-
+- Node.js 18+ (or 20+ recommended)
+- One package manager (we use `pnpm` in this epo)
 
+### Setup
+```bash
+cd frontend
+pnpm install
+cp .env.local.example .env.local   # fill alues below
+```
+
+Required env vars:
+- NEXT_PUBLIC_API_URL=http://ocalhost:8080        # API Gateway base URL
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)